src/site/stages/build/process-cms-exports/transformers/node-story_listing.js

```diff
@@ -10,26 +11,52 @@ const transform = entity => ({
   title: getDrupalValue(entity.title),
   created: utcToEpochTime(getDrupalValue(entity.created)),
   changed: utcToEpochTime(getDrupalValue(entity.changed)),
+  entityPublished: isPublished(getDrupalValue(entity.moderationState)),
   entityMetatags: createMetaTagArray(entity.metatag.value),
   fieldAdministration: entity.fieldAdministration[0],
   fieldDescription: getDrupalValue(entity.fieldDescription),
   fieldIntroText: getDrupalValue(entity.fieldIntroText),
   fieldMetaTitle: getDrupalValue(entity.fieldMetaTitle),
   fieldOffice: entity.fieldOffice[0],
+  reverseFieldListingNode: {
+    entities: entity.reverseFieldList
+      .filter(
+        reverseField =>
+          reverseField.entityBundle === 'news_story' && reverseField.status,
```

Comment:
Alternatively, you could add `entityPublished` to `node-news_story` and use that. Instead of `reverseField.status`, it'd be `reverseField.isPublished` which is easier to grok.